charts/skypilot/values.yaml

```diff
@@ -196,6 +196,13 @@ rbac:
     - apiGroups: [ "" ]
       resources: [ "configmaps" ]
       verbs: [ "get", "patch" ]
+    # Required for high-availability controller
```

Comment:
This applies to both serve and jobs controller, I assume. We were just missing it before?